config/relay.go

```diff
@@ -30,6 +31,11 @@ func (r *Relay) Default() error {
 		r.ListenTimeout = 15
 	}
 
+	if r.ListenTimeout >= 30 {
+		err := fmt.Errorf("listen_timeout should be less 30s (APIGateway integration limit)")
+		return errors.Wrap(err, ".listen_timeout")
```

Comment:
Not sure the vision here, but it did feel a little dirty to assume APIGateway in this part of the code.

Reply:
We could move that to a `Validate()` method in `./config/lambda.go`, I suppose we should give it a little breathing room to ensure the error surfaces, API GW-level timeouts are a bit more annoying to troubleshoot

I didn't realize API GW's was so low, would definitely be nice to cap `lambda.timeout` to that as well.

Reply:
Hmm, I couldn't really find a great way to accomplish that without an import cycle.  I went ahead and made `Relay` aware of what platform it is deployed on, so that it can validate itself based on that.